spec/mongoid/document_spec.rb

```diff
@@ -339,23 +318,6 @@ class << self; attr_accessor :name; end
 
   describe ".instantiate" do
 
-    context "when passing a block" do
-
-      let(:id) do
-        BSON::ObjectId.new
-      end
-
-      let(:document) do
-        Band.instantiate("_id" => id, "name" => "Depeche Mode") do |band|
-          band.likes = 1000
-        end
-      end
-
-      it "yields to the block" do
-        expect(document.likes).to eq(1000)
```

Comment:
I can't find this test in the green part of the diff, is it in here somewhere?

Reply:
I have added this test back, as well as the functionality for giving a block to instantiate.

Reply:
That was added in 9c4daf4435, not clear to what end. Given the test added in that commit perhaps the goal of the block API is to permit augmenting the created object during instantiation rather than consuming it in the block. Let's keep the block API for now.

Reply:
Okay great